.github/workflows/build-storybook.yml

```diff
@@ -47,3 +47,42 @@ jobs:
           GIT_AUTHOR_NAME: ${{ github.actor }}
           GIT_COMMITTER_EMAIL: ${{ github.actor }}@users.noreply.github.com
           GIT_COMMITTER_NAME: ${{ github.actor }}
+
+  create-storybook-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find Comment
```

Comment:
Can we try to put `find-comment` in a block and re-use it since we are duplicating the code. You can have a look at https://stackoverflow.com/questions/8466223/reuse-a-block-of-code-in-yaml

Reply:
thanks for the suggestion @osamasayed 👍 

we found a way to share the output between jobs, so use that now.